c++/src/OrcHdfsFile.cc

```diff
@@ -34,15 +34,13 @@
 #include "common/hdfs_configuration.h"
 #include "common/configuration_loader.h"
 
-#define BUF_SIZE 1048576 //1 MB
-
 namespace orc {
 
   class HdfsFileInputStream : public InputStream {
```

Comment:
Does HDFS C++ provide a mocked service interface? My concern is the testability of HdfsFileInputStream.